apps/condo/domains/organization/integrations/sbbol/utils/changeClientSecret.js

```diff
@@ -2,15 +2,29 @@ const querystring = require('querystring')
 const dayjs = require('dayjs')
 const conf = require('@open-condo/config')
 const { SbbolRequestApi } = require('../SbbolRequestApi')
+const { SBBOL_IMPORT_NAME } = require('@condo/domains/organization/integrations/sbbol/constants')
 const { getOrganizationAccessToken } = require('./getOrganizationAccessToken')
 const { getSbbolSecretStorage } = require('./getSbbolSecretStorage')
+const { getSchemaCtx } = require('@open-condo/keystone/schema')
+const { User } = require('@condo/domains/user/utils/serverSchema')
+const { Organization } = require('@condo/domains/organization/utils/serverSchema')
 
 const SBBOL_FINTECH_CONFIG = conf.SBBOL_FINTECH_CONFIG ? JSON.parse(conf.SBBOL_FINTECH_CONFIG) : {}
 const SBBOL_PFX = conf.SBBOL_PFX ? JSON.parse(conf.SBBOL_PFX) : {}
 
-async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret }) {
-    const accessToken = await getOrganizationAccessToken()
+async function changeClientSecret ({ clientId, currentClientSecret, newClientSecret, userId }) {
     const sbbolSecretStorage = getSbbolSecretStorage()
+    const { keystone: userContext } = await getSchemaCtx('User')
+
+    const organization = await Organization.getOne(userContext, { importId: SBBOL_FINTECH_CONFIG.service_organization_hashOrgId } )
+    if (!organization) {
+        throw new Error('Could not fetch SBBOL service organization by importId with corresponds to SBBOL_FINTECH_CONFIG.service_organization_hashOrgId')
+    }
+    const user = await User.getOne(userContext, { organization: { id: organization.id }, deletedAt: null })
```

Comment:
To avoid fetching non-admin users, we can fetch first organization employee and then access token of its user